airbyte-integrations/connectors/destination-deepset/pyproject.toml

```diff
@@ -0,0 +1,28 @@
+[build-system]
+requires = ["poetry-core>=1.0.0"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry]
+version = "0.1.0"
+name = "destination-deepset"
+description = "Destination implementation for deepset."
+authors = ["Airbyte <contact@airbyte.io>"]
+license = "MIT"
+readme = "README.md"
+documentation = "https://docs.airbyte.com/integrations/destinations/deepset"
+homepage = "https://airbyte.com"
+repository = "https://github.com/airbytehq/airbyte"
+packages = [{ include = "destination_deepset" }, { include = "main.py" }]
+
+[tool.poetry.dependencies]
+python = "^3.9,<3.12"
+airbyte-cdk = "^0"
+httpx = "^0.28.1"
+tenacity = "^9.0.0"
+
+[tool.poetry.scripts]
+destination-deepset = "destination_deepset.run:run"
+
+[tool.poetry.group.dev.dependencies]
+pytest = "*"
```

Comment:
Nit: we might want to pin ^8 to make sure it does not somehow resolve do under 6.